manifest.toml

```diff
@@ -45,8 +45,8 @@ ram.runtime = "50M"
 
 [resources]
     [resources.sources.main]
-    url = "https://github.com/cryptpad/cryptpad/archive/refs/tags/2024.3.1.tar.gz"
-    sha256 = "9149c55f09b245cde2d295efe1f373c394db7a7652ba32432d05dc11c2ddb697"
+    url = "https://github.com/xwiki-labs/cryptpad/archive/refs/tags/2024.6.0.tar.gz"
```

Comment:
It is because the auto-updater checks the `testing` branch, which still contains the old URL in its `manifest.toml`. It should not happen anymore after we merge this :)